frontend/src/properties/registration.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Helper texts meant to guide the user to fill out
+ * forms on the application.
+ */
+const helperTexts = {
```

Comment:
Nice

Reply:
Thank you!

A JSDoc comment should start with /** and end with */ so it is a valid JSDoc, but in its simplest form. I export it, so I consider it to be public, as it can be imported in any other file.